### PR TITLE
feat: add troubleshooting for slow uploads

### PR DESCRIPTION
**Pull request by GaryHomewood**
Added section on troubleshooting page for slow uploads.

### CODE REVIEW

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Docusaurus build was successful!

File: docs/how-tos/troubleshooting.md [hunk 1]
```diff
@@ -38,6 +38,10 @@ The HTTP API imposes rate limits to ensure that a single user cannot overwhelm t
 
 Rate limits are imposed when more than 30 requests from the same API token are received within a ten second window. To avoid being limited, try to throttle your requests to stay within this limit. Alternatively, you can respond to a 429 status by backing off for a few seconds and retrying the request.
 
+## I am experiencing slowness in uploads to web3.storage
+
+Please make sure that you are using the latest version of web3.storage client: 3.5.6. There is an issue in the latest Node version ([nodejs/node#42117](https://github.com/nodejs/node/pull/42117)) and we reverted using native version of Blob in latest release until that fix is shipped.
```
Comment (alanshaw) [inline docs/how-tos/troubleshooting.md:43]:
```suggestion
Please make sure that you are using the latest version of web3.storage client: v3.5.6 or greater. There is an issue in the latest Node version ([nodejs/node#42117](https://github.com/nodejs/node/pull/42117)) and we reverted using native version of Blob in latest release until that fix is shipped.
```

**Review by alanshaw — APPROVED** [review]

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Docusaurus build was successful!

File: docs/how-tos/troubleshooting.md (hunk 1 above)
Comment (GaryHomewood) [reply docs/how-tos/troubleshooting.md]: Apologies, I had missed this change request. Now done.

**Pull request merged**